docs/concepts.rst

```diff
@@ -584,6 +584,8 @@ object types it's more efficient or more idiomatic to use a different format.
 There are also some types of objects that can't be pickled at all.  Bionic uses
 ``pickle`` by default, but handles some types specially:
 
+- `JSON <https://www.json.org/json-en.html>`_-serializable built-in types are
```

Comment:
Sorry I missed this in our Slack conversation, but we should probably be explicit about what "JSON-serializable" means here.